src/taxonomy/TaxonomyListPage.jsx

```diff
@@ -157,6 +158,7 @@ const TaxonomyListPage = () => {
     return { taxonomyListData, isLoaded };
   };
   const { taxonomyListData, isLoaded } = useTaxonomyListData();
+  const canAdd = isLoaded ? taxonomyListData.canAdd : false;
```

Comment:
Same. I think we need the canTagObject flag here

Reply:
I disagree -- this is `taxonomy.canAdd` , which indicates whether the user can create a new taxonomy.

But clearly these permissions are confusing, so I can rename them to include the model name:

* objecttag.canAdd -> `canTagObjects` (already done)
* objecttag.canDelete -> `canDeleteObjectTag`
* taxonomy.canAdd -> `canAddTaxonomies`
* tag.canChange -> `canChangeTag`

Reply:
My bad here @pomegranited! You're right. I meant to mark the canAdd from ObjectTag.

Reply:
No worries at all @rpenido ! It really helped me to hear your perspective reviewing these PRs, and I hope the code is now more legible to everyone.